package.json

```
@@ -56,7 +56,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.12.1",
     "@fortawesome/react-fontawesome": "^0.1.5",
     "@storybook/addon-storysource": "^5.3.18",
-    "align-diarized-text": "^1.0.7",
+    "align-diarized-text": "^1.0.8",
```

Latest version with your changes it's version 9, but happy to change that later.
```suggestion
    "align-diarized-text": "^1.0.9",
```